### DISCUSSION TITLE
Use Tailwind CSS?

### DISCUSSION THREAD

**Discussion by tianheg** [yihong0618/running_page#647] [Ideas]
I have migrated from Tachyons CSS to TailwindCSS, as tachyons-sass has not been updated for three years. Can I submit a Pull Request (PR) to change the styles? @yihong0618 

Here is the effect: https://run.tianheg.org/

**Comment by yihong0618** [comment]
of course. gogogo

**Comment by yihong0618** [comment]
Done
